OPHD/States/CrimeRateUpdate.cpp

```diff
@@ -5,14 +5,16 @@
 #include <NAS2D/Utility.h>
 
 
-int CrimeRateUpdate::update(const std::vector<TileList>& policeOverlays, PopulationPanel& populationPanel)
+void CrimeRateUpdate::update(const std::vector<TileList>& policeOverlays, PopulationPanel& populationPanel)
 {
+	mMoraleChange = 0;
+
 	const auto& structuresWithCrime = NAS2D::Utility<StructureManager>::get().structuresWithCrime();
 
 	// Colony will not have a crime rate until at least one structure that supports crime is built
 	if (structuresWithCrime.empty()) {
```

Comment:
Another minor nitpick, the general styling here should be brace placement on a new line. Not going to complain too loudly but this is the preferred style choice for the project and for the sake of consistency. :D

Reply:
I knew this was the general style you were going for, but I guess I forgot or missed these lines. Will fix momentarily.